apps.yaml

```
@@ -201,12 +201,15 @@ FullProfAPP:
         description: |
             An PyQt5 based graphical user interface for the automated analysis of powder X-ray diffraction data based on the refinement engine FullProf.
         authors: Oier Arcelus, Juan Rodriguez-Carvajal, Nebil Ayape Katcho
-        external_url: https://code.ill.fr/rodriguez-carvajal/fullprof-app
-        logo: https://code.ill.fr/rodriguez-carvajal/fullprof-app/-/raw/master/src/fpgui/icons/splash.png
+        external_url: https://www.ill.eu/sites/fullprof/php/downloads.html
+        documentation_url: https://fullprofapp.readthedocs.io/en/stable/
+        logo: https://gitlab.com/d7081/fullprofapp/-/tree/main/src/fpgui/icons/splash.png
         state: development
+        version: 1.0
```

This needs to be a string, so somethign like:
```suggestion
        version: '1.0'
```